CondFormats/JetMETObjects/interface/JetCorrectorParameters.h

```diff
@@ -149,8 +151,6 @@ class JetCorrectorParametersCollection {
 		 UncertaintyPileUpJetRate=26,
 		 UncertaintyAux1=34,
 		 UncertaintyAux2=35,
-		 UncertaintyAux3=36,
```

Comment:
Alexx, you have to put these two (36 and 37) back. You just change "UncertaintyAux1" to "L1RC" and "UncertaintyAux2" to "L1Residual".